### PR TITLE
[#5467] Make app searchable with Spotlight

### LINKED ISSUE #5467: `StatusIm` application couldn't be found in Spotlight Search
### User Story

As a user, I want to navigate to StausIm through spotlight search

### Description

*Type*: Bug
*Summary*: when I have app installed (copied to`Applications` folder) and try to look it using Spotlight Search, but `StatusIm` app is not suggested so I need manually open `Applications` folder and launch app.

#### Expected behavior
`StatusIm` is suggested, i.e. as `Chrome`
tequilajazzz 2018-08-13 10-44-18

#### Actual behavior
`StatusIm` is not suggested
tequilajazzz 2018-08-13 10-45-52

### Reproduction

- Download `.dmg` file 
- Open it
- Drag'n'drop `StatusIm` to `Applications` folder
- Open Spotlight Search
- Start type "Status"

### Additional Information
* Status version: nightly 13/08/2018
* Operating System: Mac OSx High Sierra

### PR DESCRIPTION
**Pull request by gnl**
Add metadata to `Info.plist` to allow application bundle to be properly indexed by Spotlight. Fixes #5467. 

Before:
screen shot 2018-08-17 at 11 40 12

After:
screen shot 2018-08-17 at 20 01 40

### CODE REVIEW

File: ci/Jenkinsfile.desktopbuild [hunk 1]
```diff
@@ -133,7 +133,7 @@ timeout(90) {
 
               stage('Create MacOS Bundle') {
                 dir(packageFolder) {
-                  sh 'git clone https://github.com/vkjr/StatusAppFiles.git'
+                  sh 'git clone https://github.com/gnl/StatusAppFiles.git'
```
Comment (flexsurfer) [inline ci/Jenkinsfile.desktopbuild]: why not in status repo?

**Review by goranjovic — APPROVED** [review]

**Comment by vkjr** [comment]
@gnl, could you please put `[WIP]` in a header of this PR if it is not ready and will be reimplemented.
Thanks :)

**Comment by gnl** [comment]
@vkjr No need, it's good to go, squashed and signed.

**Review by vkjr — APPROVED** [review]

**Comment by churik** [comment]
Doesn't affect mobile builds;
Tested (Mac OSx):
- installation
- log in
- spotlight search (suggestion)
- open app from suggestions

**Pull request merged**